.github/workflows/ci.yml

```diff
@@ -1,13 +1,18 @@
-name: Valhalla Changelog Validation
+name: GitHub CI
```

Comment:
nit: for visibility reasons I'd prefer to see a separate action in the PR overview, so that I know at a glance it's changelog or spell check failing. 2 jobs would do. you can leave the name here, but I'd ask to break out the step out to make it a job.